### DISCUSSION TITLE
Last steps

### DISCUSSION THREAD

**Discussion by redblindbananaz** [The-Digital-Dilemma/GetIT#55] [Announcements]
Thank you for everyone that participate to this project.
It is time now, to close up all branches and have a last test before submitting.
@charlie-canning-whitecliffe , could you please finish that footer by 5pm, so we can review and adapt the changes?

Once everyone is happy with the result, I'll finish tidy up the project and read me file then I will copy the folder for submission.

**Comment by redblindbananaz** [comment]
@Charles @makikamiya @brilliant-light @shaheensystems

**Comment by makikamiya** [comment]
Thank you for looking after the project closure and for all your awesome work, @redblindbananaz . Indeed, thank you to everyone for all your collaborative efforts on this project. I think the web app looks ABSOLUTELY BRILLIANT!!!

**Comment by redblindbananaz** [comment]
Thank you @makikamiya , after the last merge, I will copy the repo and zip all our documentation for submission and message our team on MS Team.
